.pre-commit-config.yaml

```
@@ -0,0 +1,16 @@
+repos:
+  - repo: https://github.com/sphinx-contrib/sphinx-lint
+    rev: v0.6.8
+    hooks:
+      - id: sphinx-lint
+
+  - repo: https://github.com/psf/black
```

Using this mirror lets us use mypyc-compiled black, which is about 2x faster:

https://black.readthedocs.io/en/stable/integrations/source_version_control.html

```suggestion
  - repo: https://github.com/psf/black-pre-commit-mirror
```

---

Thanks! Didn't know this existed.